gustaf/io/mfem.py

```diff
@@ -198,6 +203,62 @@ def format_array(array):
 
     # Export 3D mesh
     else:
-        raise NotImplementedError(
-            f"Sorry, we cannot export mesh of dimension {dim}."
+        # Elements
+        element_attribute = 1  # Other numbers not yet supported
+        elements = mesh.elements
+        n_elements, n_element_vertices = elements.shape
+        if n_element_vertices == 4:
```

Comment:
`mesh.whatami` gives a keyword of what it is and I see that you extract `faces()` anyways, so you can use `faces.whatami` too

Reply:
This is not strictly necessary for this PR, but I'd appreciate if you can split 2d and 3d parts into a internal function like `def _export_2d()`, which returns element / boundary / vertices string